diff -ruN brag-1.4.0.orig/Makefile brag-1.4.0/Makefile
--- brag-1.4.0.orig/Makefile	2003-11-24 12:04:41.949594680 +0100
+++ brag-1.4.0/Makefile	2003-11-24 12:12:59.983881960 +0100
@@ -7,7 +7,7 @@
 
 DST=$(ROOT)/usr
 BINDIR=$(DST)/bin
-MANDIR=$(DST)/man
+MANDIR=$(DST)/share/man
 DOCDIR=$(DST)/share/doc/brag-$(VERSION)
 RPMBASEDIR=/usr/src/redhat
 
@@ -16,10 +16,9 @@
 all:
 
 install:
-	$(INSTALL) -m 755 -d $(BINDIR) $(MANDIR)/man1 $(DOCDIR)/brag
-	$(INSTALL) -m 755 -o 0 -g 0 brag $(BINDIR)
-	$(INSTALL) -m 644 -o 0 -g 0 brag.1 $(MANDIR)/man1
-	$(INSTALL) -m 644 -o 0 -g 0 CHANGES LICENSE README $(DOCDIR)/brag
+	$(INSTALL) -m 755 -d $(BINDIR) $(MANDIR)/man1 
+	$(INSTALL) -m 755 brag $(BINDIR)
+	$(INSTALL) -m 644 brag.1 $(MANDIR)/man1
 
 clean:
 	find . -name '*~' -type f | xargs rm -f
diff -ruN brag-1.4.0.orig/Makefile~ brag-1.4.0/Makefile~
--- brag-1.4.0.orig/Makefile~	1970-01-01 01:00:00.000000000 +0100
+++ brag-1.4.0/Makefile~	2003-11-24 12:08:20.799324464 +0100
@@ -0,0 +1,52 @@
+#
+# Name		Makefile
+# Version	$Id$
+#
+
+VERSION=1.4.0
+
+DST=$(ROOT)/usr
+BINDIR=$(DST)/bin
+MANDIR=$(DST)/share/man
+DOCDIR=$(DST)/share/doc/brag-$(VERSION)
+RPMBASEDIR=/usr/src/redhat
+
+INSTALL=install
+
+all:
+
+install:
+	$(INSTALL) -m 755 -d $(BINDIR) $(MANDIR)/man1 
+	$(INSTALL) -m 755 -o 0 -g 0 brag $(BINDIR)
+	$(INSTALL) -m 644 -o 0 -g 0 brag.1 $(MANDIR)/man1
+
+clean:
+	find . -name '*~' -type f | xargs rm -f
+	rm -f brag-$(VERSION).tar.gz
+	rm -rf rpm
+
+rpm: tar brag.spec
+	mkdir -p $(RPMBASEDIR)/SOURCES
+	cp -f brag-rpm.xpm $(RPMBASEDIR)/SOURCES
+	mv -f brag-$(VERSION).tar.gz $(RPMBASEDIR)/SOURCES
+	rm -f sources/brag-$(VERSION).spec
+	sed 's/__VERSION__/$(VERSION)/g' brag.spec \
+	  > $(RPMBASEDIR)/SOURCES/brag-$(VERSION).spec
+	rpmbuild -ba --target noarch $(RPMBASEDIR)/SOURCES/brag-$(VERSION).spec
+	cd $(RPMBASEDIR)/SOURCES; \
+	  rm -f brag-$(VERSION).tar.gz brag-rpm.xpm brag-$(VERSION).spec
+
+tar: clean
+	cd ..; \
+	  rm -f brag-$(VERSION); \
+	  ln -s brag brag-$(VERSION); \
+	  tar czf brag/brag-$(VERSION).tar.gz brag-$(VERSION)/*; \
+	  rm -f brag-$(VERSION)
+
+backup: tar
+	mkdir -p $(HOME)/backup
+	mv -f brag-$(VERSION).tar.gz $(HOME)/backup
+
+#
+# End		Makefile
+#
